src/main/java/seedu/address/model/book/BookNameContainsKeywordsPredicate.java

```diff
@@ -18,7 +17,7 @@ public BookNameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Book book) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(book.getBookName().fullBookName, keyword));
+                .anyMatch(keyword -> book.getBookName().toString().toUpperCase().contains(keyword.toUpperCase()));
```

Comment:
Good use of streams, but perhaps can use StringUtil methods to avoid code duplication